pkg/syncer/util.go

```diff
@@ -133,3 +134,28 @@ func getPVCKey(ctx context.Context, obj interface{}) (string, error) {
 	log.Infof("getPVCKey: PVC key %s", objKey)
 	return objKey, nil
 }
+
+// GetVolumeID returns volume handle if successfully found by the volumeMigrationService
+func GetVolumeID(ctx context.Context, volumePath string) (string, error) {
+	log := logger.GetLogger(ctx)
+	volumeHandle, err := volumeMigrationService.GetVolumeID(ctx, volumePath)
+	if err != nil {
+		log.Errorf("failed to get VolumeID from volumeMigrationService for volumePath: %q", volumePath)
+		return "", err
+	}
+	return volumeHandle, nil
+}
+
+// HasMigratedToAnnotation returns true if the migrated-to annotation is found in the newer object
+func HasMigratedToAnnotation(ctx context.Context, prevAnnotations map[string]string, newAnnotations map[string]string) bool {
+	log := logger.GetLogger(ctx)
+	// Checking if the migrated-to annotation is found in the new PV
+	if _, annMigratedToFound := newAnnotations[common.AnnMigratedTo]; annMigratedToFound {
+		if _, annMigratedToFound = prevAnnotations[common.AnnMigratedTo]; !annMigratedToFound {
+			log.Debugf("Received %v annotation update", common.AnnMigratedTo)
+			return true
+		}
+	}
+	log.Debug("%v annotation not found", common.AnnMigratedTo)
```

Comment:
This message is for which PV or PVC, not clear. As mentioned in the previous comment, can we add info level log where this func is called?

Reply:
Added.